src/gluonts/model/seq2seq/_forking_network.py

```diff
@@ -281,3 +351,53 @@ def hybrid_forward(
         predictions = self.quantile_proj(fcst_output).swapaxes(2, 1)
 
         return predictions
+
+    
+class ForkingSeq2SeqDistributionNetwork(ForkingSeq2SeqNetworkBase):
```

Comment:
Maybe a better name for this (although longer) is

```suggestion
class ForkingSeq2SeqDistributionPredictionNetwork(ForkingSeq2SeqNetworkBase):
```